.golangci.yml

```diff
@@ -18,3 +18,7 @@ issues:
 linters-settings:
   errcheck:
     exclude: scripts/errcheck_excludes.txt
+
+issues:
+  exclude:
+    - composite
```

Comment:
why is it needed?

Reply:
This is required, so that CI stops complaining on wrapping Prometheus object to PrometheusServer. Eg. `operator.PrometheusServer(&promv1.Prometheus)` is being signaled by govet [1].

[1] https://github.com/alessio/golangci-lint/commit/68b8aaacd09b1591ddc982d5dfa5815f19b214a2